### PR TITLE
5.1.2.1

### PR DESCRIPTION
**Pull request by artur-stepien**

### CODE REVIEW

File: joomla_v5/translations/package/pl-PL/pkg_pl-PL.xml [hunk 1]
```diff
@@ -14,11 +14,11 @@
 	<packagerurl>https://www.joomla.pl</packagerurl>
 	<description>	<![CDATA[
 		<div class="text-center span12"> 
-		<h3>Joomla! - pakiet języka polskiego - 5.0.3</h3>
+		<h3>Joomla! - pakiet języka polskiego - 5.1.1</h3>
```
Comment (tecpromotion) [inline joomla_v5/translations/package/pl-PL/pkg_pl-PL.xml:17]:
```suggestion
		<h3>Joomla! - pakiet języka polskiego - 5.1.2</h3>
```

**Review by tecpromotion — CHANGES_REQUESTED** [review]
Please check version

**Comment by artur-stepien** [comment]
> Thanks @artur-stepien

Build systems seems to be a bit off. I will rewrite it to `phing` just need to find some free time.

**Comment by tecpromotion** [comment]
> > Thanks @artur-stepien
> 
> Build systems seems to be a bit off. I will rewrite it to `phing` just need to find some free time.

What exactly do you mean by that?
The build scripts run perfectly. If you want to join me, please feel free to contact me on Mattermost.

**Pull request merged**